packages/sanity/src/core/versions/components/dialog/BundleForm.tsx

```diff
@@ -0,0 +1,136 @@
+/* eslint-disable i18next/no-literal-string */
+import {CalendarIcon} from '@sanity/icons'
+import {
+  Box,
+  Button,
+  type ButtonTone,
+  Card,
+  Flex,
+  Select,
+  Stack,
+  Text,
+  TextArea,
+  TextInput,
+} from '@sanity/ui'
+import {useCallback} from 'react'
+
+import {type Bundle} from '../../types'
+import {RANDOM_TONES} from '../../util/const'
+import {toSlug} from '../../util/dummyGetters'
+
+export function BundleForm(props: {
```

Comment:
I'm thinking if we should add some validations here, and also on the store, to disallow the creation of bundles with the name `draft` or `drafts`

Reply:
Very good point, adding it 👍